components/inputs/input-number.js

```diff
@@ -22,6 +22,7 @@ class InputNumber extends SkeletonMixin(FormElementMixin(LocalizeCoreElement(Lit
 			autocomplete: { type: String },
 			autofocus: { type: Boolean },
 			disabled: { type: Boolean },
+			hideInvalidIcon: { attribute: 'hide-invalid-icon', type: Boolean, reflect: true },
```

Comment:
Needed to expose these things on `d2l-input-number` so that `d2l-input-percent` could use them.